bin/apt-dependencies.sh

```diff
@@ -118,8 +118,8 @@ EOF
 fi
 
 # rest of python dependencies
-if [ "${VERSION_CODENAME}" == "bookworm" ]; then
-    # On Debian bookworm, need the --break-system-package to into to default system location
+if [ "${VERSION_CODENAME}" == "bookworm" -o "${VERSION_CODENAME}" == "noble" ]; then
+    # On Debian bookworm and Ubuntu noble, need the --break-system-package to into to default system location
     pip3 --default-timeout=10000 install --break-system-packages --upgrade sphinx_rtd_theme nose requests hypothesis==3.79.0
```

Comment:
@nickva Do we really need this anymore (also applies to the other CI images). I think we have our own venv in the docs folder for this, when building the docs ...

Reply:
it fails without it, seems we don't make a venv where we should.